lib/dataElementListener.js

```diff
@@ -372,6 +372,14 @@ class DataElementImporter extends SHRDataElementParserListener {
       } else if (cst.elementStringConstraint()) {
         const str = stripDelimitersFromToken(cst.elementStringConstraint().STRING());
         value.addConstraint(new FixedValueConstraint(str, 'string', path));
+      } else if (cst.elementIntegerConstraint()) {
+        const minus = cst.elementIntegerConstraint().MINUS() ? '-' : '';
+        const num = cst.elementIntegerConstraint().WHOLE_NUMBER().getText();
+        const int = parseInt(minus + num);
+        value.addConstraint(new FixedValueConstraint(int, 'integer', path));
+      } else if (cst.elementDecimalConstraint()) {
+        const decimal = parseFloat(cst.elementDecimalConstraint().getText().substr(1));
```

Comment:
This works, but I had to look at the grammar to remember what character the `substr(1)` was skipping.  Could you add a comment that indicates the `substr(1)` is there to skip the `=`?